decimal64math.go

```diff
@@ -124,6 +124,52 @@ func (d Decimal64) Sqrt() Decimal64 {
 	exp, significand = renormalize(exp/2-8, sqrt)
 	return newFromParts(sign, exp, significand)
 }
+func (dp *decParts) round128to64() discardedDigit {
+	var rndStatus discardedDigit
+	if dp.significand.numDecimalDigits() > 16 {
+		var remainder uint64
+		expDiff := dp.significand.numDecimalDigits() - 16
+		dp.exp += expDiff
+		dp.significand, remainder = dp.significand.divrem64(powersOf10[expDiff])
+		rndStatus = roundStatus(remainder, 0, expDiff)
+	}
+	// dp.significand.lo = dp.significand.lo
+	return rndStatus
+}
+
+func (dp *decParts) Add128(ep *decParts) decParts {
```

Comment:
Could you add some short comments to explain these different if statements?

Reply:
for Add128, just add a brief introduction into what this function does, e.g. why is it 128 instead of 64 and what will this try to do?